pkg/aria2/common/common.go

```diff
@@ -38,6 +38,8 @@ const (
 	Downloading
 	// Paused 暂停中
 	Paused
+	// Seeding 做种中
```

Comment:
In my opinion, we'd better move the newly added status enums to the end.
Otherwise, it may cause some conflicts in programs which saved enums' values.

For example, enum value `5` means Complete before, but after the upgrade `5` means Error.

Or maybe I'm overthinking 😃 , correct me, please.

Reply:
It should be less than error, because https://github.com/cloudreve/Cloudreve/blob/96daed26b4453093d9d1a4dfe8d9ee334a7338af/service/aria2/manage.go#L60

Reply:
You're right, it's my wrong. 😅 
I'm looking forward to the merge!

Reply:
> It should be less than error, because
> 
> https://github.com/cloudreve/Cloudreve/blob/96daed26b4453093d9d1a4dfe8d9ee334a7338af/service/aria2/manage.go#L60

We should append new enum to the end as @topjohncian suggested, and refactor the [line#60](https://github.com/cloudreve/Cloudreve/blob/96daed26b4453093d9d1a4dfe8d9ee334a7338af/service/aria2/manage.go#L60).
Inserting new enum in the middle will break backwards compatibility:  Download record in database from old version with be mapped to wrong enum in new version.

Reply:
Done